desktop/core/src/desktop/auth/api_authentications.py

```diff
@@ -81,3 +81,16 @@ def authenticate(self, request):
       user.profile.save()
 
     return (user, None)
+
+
+# for local dev env doesn't have authentication service
```

Comment:
niy: Typo? and upper case letter?

Reply:
hint: usually better to but those comments as comment on the line after the class name:

```
class DummyCustomAuthentication(authentication.BaseAuthentication):
"""
...
"""
```